app/scripts/modules/kubernetes/mapConfigurer/mapConfigurer.component.ts

```diff
@@ -0,0 +1,63 @@
+import {module} from 'angular';
```

Comment:
Please change the formatting to `import { module }` here and below. (That's the new style, though it's not well reflected in the codebase right now).